### ISSUE TITLE
Streaming

### ISSUE THREAD

**Issue by remexre** [asciinema/asciinema-player#123]
Currently, it looks like the player requires the entirety of the screencast to be available; I'd like to use this for real-time streaming, though.

Would it be plausible possible to add a streaming API, to allow appending to the screencast loaded by an element? I'd likely be hooking this up to either an [EventSource](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events) or WebSocket.

**Comment by sickill** [comment]
Definitely! I'm working on the next generation of the player, and one of the goals there is to have easy streaming capability (with both EventSource and WebSocket options).

**Comment by neilmock** [comment]
There appears to be an EventSource Source implemented, is it possible to use this implementation instead of Recording, via the `type` option (which defaults to asciicast currently)?

**Comment by TACIXAT** [comment]
How is this going? I would be happy to help if it is simple enough. 

I'll be receiving a handful of different events over websocket and am looking for a way to send the asciinema ones to the player.

**Comment by mixmix** [comment]
I came here after looking for something similar and seeing this blog post which describes streaming compatability of new file format with net at example

https://blog.asciinema.org/post/two-point-o/
